isis/src/base/apps/noproj/main.cpp

```diff
@@ -1,440 +1,13 @@
-#define GUIHELPERS
 #include "Isis.h"
 
-#include <iostream>
-#include <sstream>
-#include <QString>
+#include "noproj.h"
 
-#include "AlphaCube.h"
 #include "Application.h"
-#include "Blob.h"
-#include "CameraDetectorMap.h"
-#include "CameraFocalPlaneMap.h"
-#include "History.h"
-#include "iTime.h"
-#include "Process.h"
-#include "ProgramLauncher.h"
-#include "Pvl.h"
-#include "PvlObject.h"
 
-
-using namespace Isis;
 using namespace std;
-
-void LoadMatchSummingMode();
-void LoadInputSummingMode();
-
-map <QString, void *> GuiHelpers() {
-  map <QString, void *> helper;
-  helper ["LoadMatchSummingMode"] = (void *) LoadMatchSummingMode;
-  helper ["LoadInputSummingMode"] = (void *) LoadInputSummingMode;
-  return helper;
-}
```

Comment:
Removed this in noproj.cpp since it appears unused and when running noproj through the GUI with this removed, gave the same output as with it.

Reply:
Guihelpers placed back into noproj